front/src/components/boxs/device-in-room/EditDevices.jsx

```diff
@@ -0,0 +1,118 @@
+import { Component } from 'preact';
+import { Localizer, Text } from 'preact-i18n';
+import { connect } from 'unistore/preact';
+import Select from 'react-select';
+import BaseEditBox from '../baseEditBox';
+import { getDeviceFeatureName } from '../../../utils/device';
+import withIntlAsProp from '../../../utils/withIntlAsProp';
+import SUPPORTED_FEATURE_TYPES from './SupportedFeatureTypes';
+
+class EditDevices extends Component {
+  updateDeviceFeatures = selectedDeviceFeaturesOptions => {
+    selectedDeviceFeaturesOptions = selectedDeviceFeaturesOptions || [];
+    const deviceFeatures = selectedDeviceFeaturesOptions.map(option => option.value);
+    this.props.updateBoxConfig(this.props.x, this.props.y, {
+      device_features: deviceFeatures
+    });
+    this.setState({ selectedDeviceFeaturesOptions });
+  };
+
+  updateName = e => {
+    this.props.updateBoxConfig(this.props.x, this.props.y, {
+      name: e.target.value
+    });
+  };
+
+  getDeviceFeatures = async () => {
+    try {
+      this.setState({ loading: true });
+      // we get the rooms with the devices
+      const devices = await this.props.httpClient.get(`/api/v1/device`);
+      const deviceOptions = [];
+      const selectedDeviceFeaturesOptions = [];
+      console.log(devices);
+
+      devices.forEach(device => {
+        const deviceFeatures = [];
+        device.features.forEach(feature => {
+          const featureOption = {
+            value: feature.selector,
+            label: getDeviceFeatureName(this.props.intl.dictionary, device, feature)
+          };
+          // for now, we only supports binary on/off and sensors
```

Comment:
are we only supporting sensors & on/off switch?

Reply:
IDK, If have copied the same behaviour as DevicesInRoom

Reply:
Indeed, this is an old comment, it can be removed